icc-api/api/iccUserApi.ts

```diff
@@ -127,6 +127,18 @@ export class iccUserApi {
       .then(doc => new models.UserDto(doc.body as JSON))
       .catch(err => this.handleError(err))
   }
+  getMatchingUsers(): Promise<Array<models.UserDto> | any> {
```

Comment:
Should be used to detect login/password duplicated user. If the number of matching user is bigger than 1 the user should be blocked and contact the helpdesk asap

Reply:
This was removed upstream: https://github.com/taktik/icure-backend/commit/8908d37a220a627dd7db914782d2c8b300d29c6e